vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Provide usage documentation for your packages
+description: Guidance for adding usage documentation to packages
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 07/10/2023
+---
+# Provide usage documentation for your packages
+
+## Overview
+
+Providing usage documentation for packages allows users to easily adopt them in their
```

Comment:
```suggestion
Providing usage documentation for ports allows users to easily adopt them in their
```

Reply:
I intentionally used the word "packages" here instead of "ports", since it is the package what you integrate in your project, not the port. 

From the original versioning spec, which I think we should make its own documentation article:

## Glossary
Some of the terms used in this document have similar meaning when discussed by the community, and because of that, they can cause confusion and ambiguity. To solve this issue, we will assign specific meaning to these terms and try to keep a consistent usage through the document.

**Library**: A piece of software (source code, binary files, documentation, license, etc.) that is intended to be reused by other software.

**Package**: A package can contain a library, a collection of libraries, build scripts, software tools, or other components necessary for its own use. The goal of vcpkg is to facilitate the installation of these packages in the user's environment.

**Port**: A vcpkg specific term, a port contains:

* Metadata about a package: package version, supported features, dependencies, etc.
* Instructions to acquire, build if necessary, and install the package.